docs/spec_v5.md

```diff
@@ -30,7 +30,7 @@ In [ABNF syntax](https://tools.ietf.org/html/rfc5234), Twirp's URLs
 have the following format:
 
 ```abnf
-URL ::= Base-URL "/twirp/" [ Package "." ] Service "/" Method
+URL ::= Base-URL [ Prefix ] "/" [ Package "." ] Service "/" Method
```

Comment:
I realize that, while this change is backwards-compatible in code, it is not backwards-compatible in the spec; the spec change means that other implementations that only allow a "/twirp" prefix are suddenly not implementing the spec properly. I guess the best path forward is to make a v6 spec. Since the previous v6 spec was archived (streams not happening), would it make sense to make a v7 spec? or is it better to make a 5.1 spec? or maybe it is fine to go with v6? @spenczar do you have a preference here? (I think either way is probably fine)

Reply:
We will be releasing this as v7, both the spec and the Twirp version. This release will contain other PRs with some other bugfixes. We will skip v6 for clarity.